src/Tabs/Tab.tsx

```diff
@@ -0,0 +1,72 @@
+import { ClassNames, css } from '@emotion/core';
+import * as React from 'react';
+import { Tab } from 'react-tabs';
+import { useTheme } from '../theme';
+
+const StyledTab: React.FunctionComponent<{ disabled?: boolean }> & { tabsRole: string } = props => {
+  const theme = useTheme();
+  const styles = tabStyles(theme.tabs);
+  return (
+    <ClassNames>
+      {({ css: getClassName }) => (
+        <Tab
+          {...props}
+          className={getClassName(styles.tabStyle)}
+          disabledClassName={getClassName(styles.disabledTabStyle)}
+          selectedClassName={getClassName(styles.selectedTabStyle)}
+        >
+          {props.children}
+        </Tab>
+      )}
+    </ClassNames>
+  );
+};
+
+StyledTab.tabsRole = 'Tab';
+
+const tabStyles = (tabTheme: { fg: string; bg: string }) => {
+  return {
+    tabStyle: css({
+      display: 'inline-block',
+      border: '1px solid transparent',
+      borderBottom: 'none',
+      bottom: '-1px',
+      position: 'relative',
+      listStyle: 'none',
+      padding: '6px 12px',
+      cursor: 'pointer',
+      backgroundColor: tabTheme.bg,
+      color: tabTheme.fg,
+
+      '&:focus': {
+        boxShadow: '0 0 5px hsl(208, 99%, 50%)',
+        borderColor: 'hsl(208, 99%, 50%)',
```

Comment:
We are missing a themed color here.
![image](https://user-images.githubusercontent.com/9273484/52274205-3e3dc680-294c-11e9-9836-e19d90a698e1.png)

You can switch between themes using our theme addon (click on theme tab)
![image](https://user-images.githubusercontent.com/9273484/52274219-4990f200-294c-11e9-9409-24c9760284ce.png)

Reply:
@XVincentX bump ^

Reply:
Apparently, it's some blue color which looks fine for both light and dark theme, so let's leave it for now.